ui/src/data-services/hooks/jobs/constants.ts

```diff
@@ -0,0 +1,2 @@
+export const COLLECTION = 'jobs'
+export const REFETCH_INTERVAL = 10000 // Refetch every 10 second
```

Comment:
@mihow here is the polling configured for jobs, you can tweak this value or set to undefined :)